### DISCUSSION TITLE
Time to first token?

### DISCUSSION THREAD

**Discussion by jrcapicua** [langfuse/langfuse#2188] [Support]
Hi guys, I'm not finding how to see in the traces the time to the first token generated?
I mean, in Generations there is a column for that, but it's always empty.
<img width="1161" alt="Captura de pantalla 2024-05-30 a la(s) 9 57 23 a  m" src="https://github.com/langfuse/langfuse/assets/46796968/7fa96c4a-1a53-47cc-9484-317ad3f8c4de">
I'm testing langfuse on a langgraph implementation, could it be that it doesn't allow to track that value?
I'm using version 2.45.1 (docker).

**Comment by marcklingen** [comment]
Do you use streaming responses? Under the hood each langfuse generation can track a `completion_start_time` ([reference](https://python.reference.langfuse.com/langfuse/client#Langfuse.generation)) which will result in this metric being computed.

Please open an issue in case this does not work for your streaming responses via langchain/langgraph on the latest Langfuse SDK version

**Comment by m1np3m** [comment]
Hi @jrcapicua Did you solve the problem ? Since I still got the same error for LLamaindex and there are no token output

**Comment by ilan-riverside** [comment]
i have a similar issue, i use langfuse with mastra ai and open ai as provider
